### ISSUE TITLE
Thermostat limits

### ISSUE THREAD

**Issue by harut1981** [ilcato/homebridge-fibaro-home-center#102]
Is it possible to change thermostat heating limit (current is 38C) to 100C? 
I need it for sauna.

P.S.
Thank you Ilcato for your excelent work!!!!

**Comment by harut1981** [comment]
Limit has changed but control is broken

**Comment by harut1981** [comment]
It works couple of times and then stops responding

**Comment by ilcato** [comment]
interesting. Any info from the log ?

**Comment by harut1981** [comment]
Hi. Nothing in logs. Don't know the reason but it works today.
Thank you.

**Comment by ilcato** [comment]
May I keep this change ?

**Comment by harut1981** [comment]
This is my first time trying to add climate zones. I think it works but can't be 100% sure.
Are you planning to add also thermostats directly, not by climate zones?

**Comment by ilcato** [comment]
No, I removed direct thermostat support because of too many different way to manage it.
